content/test/integration/controller/entity-metadata.spec.ts

```diff
@@ -1,22 +1,29 @@
 import fetch from 'node-fetch'
-import { EnvironmentConfig } from '../../../src/Environment'
 import { makeNoopValidator } from '../../helpers/service/validations/NoOpValidator'
-import { setupTestEnvironment } from '../E2ETestEnvironment'
 import { buildDeployData } from '../E2ETestUtils'
+import { createDefaultServer, resetServer } from '../simpleTestEnvironment'
+import { TestProgram } from '../TestProgram'
+import LeakDetector from 'jest-leak-detector'
 
 describe('Integration - Get wearable image and thumbnail', () => {
-  const getTestEnv = setupTestEnvironment()
-
-  it('when entity does not exist, it should return 404', async () => {
-    const server = await getTestEnv()
-      .configServer()
-      .withConfig(EnvironmentConfig.DISABLE_SYNCHRONIZATION, true)
-      .andBuild()
+  let server: TestProgram
 
+  beforeAll(async () => {
+    server = await createDefaultServer()
     makeNoopValidator(server.components)
+  })
 
-    await server.startProgram()
+  beforeEach(() => resetServer(server))
```

Comment:
and here, I hate adding these kind of repetitive comments, I'm sorry haha but maybe they are useful for you to identifying the spots faster.